lib/conjur/variable.rb

```diff
@@ -205,5 +205,19 @@ def value(version = nil)
       url << "?version=#{version}" if version
       self[url].get.body
     end
+
+    # Set the expiration for the variable.
+    #
+    # You must have the **`'update'`** permission on a variable to call this method.
+    #
+    # @example Set the expiration for a variable to tomorrow
+    #   var = api.variable 'my-secret'
+    #   var.expire 'P1D'
+    # @param duration ISO8601 duration the variable should be valid
```

Comment:
It's an ISO8601 duration, as described here: https://en.wikipedia.org/wiki/ISO_8601#Durations . It specifies how far in the future (relative to the server's notion of the current time) the variable should be set to expire.

I'll update the doc to that effect.